### PR TITLE
Feature/button

### PR DESCRIPTION
**Pull request by gopienkonikita**
Added component Button, added themes to Button component, added new tests. Updated global colors.

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import isEqual from "lodash/isEqual";
+import StyledButton from "./StyledButton";
+
+const Icon = ({ size, primary, icon }) => (
+  <div className="btnIcon">
+    {icon &&
+      React.cloneElement(icon, {
+        isfill: true,
```
Comment (AlexeySafronov) [inline src/components/Button/Button.js]: Invalid property name

File: src/components/Button/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  label="OK"
+/>
+```
+
+### Properties
+
+| Props      |   Type   | Required |         Values          | Default | Description                                           |
+| ---------- | :------: | :------: | :---------------------: | :-----: | ----------------------------------------------------- |
+| `icon`     |  `node`  |    -     |            -            | `null`  | Icon node element                                     |
+| `clicked`  |  `bool`  |    -     |            -            | `false` | Tells when the button should present a clicked state  |
```
Comment (AlexeySafronov) [inline src/components/Button/README.md]: Remove useless properties: clicked, hovered

File: src/components/Button/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+| `disabled` |  `bool`  |    -     |            -            | `false` | Tells when the button should present a disabled state |
+| `hovered`  |  `bool`  |    -     |            -            | `false` | Tells when the button should present a hovered state  |
+| `loaded`   |  `bool`  |    -     |            -            | `false` | Tells when the button should show loader icon         |
```
Comment (AlexeySafronov) [inline src/components/Button/README.md]: Rename loaded to loading

File: src/themes/base.js [hunk 4]
```diff
@@ -1,13 +1,120 @@
...
+        primaryWithLabel: "8px 24px 9px 24px",
+        primaryWithoutLabel: "8px 10px 9px 10px",
+
+        primaryWithoutIconLabel: "8px 28px 9px 28px",
+        primaryWithoutIcon: "8px 10px 9px 10px",
+
+        baseWithIconLabel: "8px 24px 9px 24px",
+        baseWithIcon: "8px 10px 9px 10px",
+
+        baseWithoutIconLabel: "8px 27px 9px 28px",
```
Comment (andreysavihin) [inline src/themes/base.js]: Why primaryWithoutIconLabel not equal baseWithoutIconLabel ?

File: src/themes/dark.js [hunk 5]
```diff
@@ -1,13 +1,116 @@
...
+        primaryWithLabel: "8px 24px 9px 24px",
+        primaryWithoutLabel: "8px 10px 9px 10px",
+
+        primaryWithoutIconLabel: "8px 28px 9px 28px",
+        primaryWithoutIcon: "8px 10px 9px 10px",
+
+        baseWithIconLabel: "8px 24px 9px 24px",
+        baseWithIcon: "8px 10px 9px 10px",
+
+        baseWithoutIconLabel: "8px 27px 9px 28px",
```
Comment (andreysavihin) [inline src/themes/dark.js]: Why primaryWithoutIconLabel not equal baseWithoutIconLabel ?

File: src/themes/base.js (hunk 4 above)
Comment (gopienkonikita) [reply src/themes/base.js]: Values according to layouts.

File: src/themes/dark.js (hunk 5 above)
Comment (gopienkonikita) [reply src/themes/dark.js]: Values according to layouts.

**Pull request merged**